dockerfiles/terraform/Dockerfile

```diff
@@ -3,7 +3,7 @@ FROM alpine:latest
 ARG VCS_REF=master
 ARG BUILD_DATE=""
 ARG REGISTRY_PATH=paritytech
-ARG TERRAFORM_VERSION="0.12.24"
+ARG TERRAFORM_VERSION="0.13.4"
```

Comment:
this is just the default version if none is specified for the build (which it is in the .gitlab-ci.yml file).

Reply:
@TriplEight do you think it would maybe make more sense to remove the version from here? seems to be confusing.

Reply:
For sure, this variable is misleading

Reply:
@gabreal 
```
 docker run -it paritytech/terraform:latest terraform --version                                                                                                                  
Terraform v0.13.4
```

Reply:
https://github.com/paritytech/scripts/pull/229